inst/htmljs/animint.js

```diff
@@ -1780,6 +1796,212 @@ var animint = function (to_select, json_file) {
 	return Selectors[s_name].duration;
       })
     ;
+    
+    // selector widgets
+    var show_message2 = "Toggle selected variables";
+    var show_hide_selector_widgets = element.append("button")
+      .text(show_message2)
+      .attr("id", "show_hide_selector_widgets")
+      .on("click", function(){
+        if(this.textContent == show_message2){
+          selector_table.style("display", "");
+          show_hide_selector_widgets.text("Hide variable toggler");
+        }else{
+          selector_table.style("display", "none");
+          show_hide_selector_widgets.text(show_message2);
+        }
+      })
+    ;
+    // adding a table for selector widgets
+    var selector_table = element.append("table")
+      .style("display", "none")
+    ;
+    var selector_first_tr = selector_table.append("tr");
+    selector_first_tr
+      .append("th")
+      .text("Toggle selected value");
+      
+    //creating an array to contain the selectized items
+    var jquery_thingy_array = [];
```

Comment:
2. If I do `var jquery_thingy_array = [];` then try to update the `vs` selector,  I get an error in line 1494 (where I modify `jquery_thingy_array` in `update_selector()`) saying that `jquery_thingy_array` is undefined.

3. If I delete the `var` and just do `jquery_thingy_array = [];` then try to update the `vs` selector, I get the error message below.  I'll keep poking around, but if anyone has an idea why this is happening, I would love to hear it.  Also, I apologize for the name `jquery_thingy_array `.  It will be updated as soon as I get this straightened out.

![image](https://cloud.githubusercontent.com/assets/6612411/9215321/0981a952-4066-11e5-96f9-6de8e3d413ed.png)